src/main/java/org/radarcns/domain/restapi/Application.java

```diff
@@ -0,0 +1,114 @@
+package org.radarcns.domain.restapi;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Application {
+
+    /**
+     * Hardware identifier of client application.
+     */
+    @JsonProperty
+    private java.lang.String ipAddress;
+    /**
+     * Time since last app start (s).
+     */
+    @JsonProperty
+    private double uptime;
+    /**
+     * Server connection status.
+     */
+    @JsonProperty
+    private ServerStatus serverStatus;
+    /**
+     * Number of records currently being cached.
+     */
+    @JsonProperty
+    private int recordsCached;
+    /**
+     * Number of records sent since application start.
+     */
+    @JsonProperty
+    private int recordsSent;
+    /**
+     * Number of unsent records.
+     */
+    @JsonProperty
+    private int recordsUnsent;
+
+    /**
+     * Default constructor.  Note that this does not initialize fields to their default values from
+     * the schema.  If that is desired then one should use <code>newBuilder()</code>.
```

Comment:
`newBuilder()` is not supported

Reply:
Perhaps it should be though... Note that Jackson is also capable of using the builder pattern using the `@JsonPOJOBuilder` and `@JsonDeserialize` annotations.